docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -4,17 +4,683 @@ title: Connector SDK
+import TabItem from "@theme/TabItem";
+
 :::note
-The **connector SDK** is in developer preview and subject to breaking changes. Use at your own risk.
+The **Connector SDK** is in developer preview and subject to breaking changes. Use at your own risk.
 :::
 
-The **connector SDK** allows you to develop your own **connectors**. With the help of the SDK you can perform common operations, such as:
+The **Connector SDK** allows you to [develop custom Connectors](#creating-a-custom-connector)
+using Java code. You can focus on the logic of the Connector, test it locally, and
+reuse its runtime logic in multiple environments. The SDK achieves this by abstracting from
+Camunda Platform 8 internals that usually come with
+[job workers](../../../concepts/job-workers.md).
+
+The SDK provides APIs for common Connector operations, such as:
+
+- Fetching and deserializing input data
+- Validating input data
+- Replacing secrets in input data
+
+Additionally, the SDK allows for convenient [testing](#testing) of your Connector behavior and
+[executing it in the environments](#runtime-environments) that suit your use cases best.
+
+## Creating a custom Connector
+
+Using the Connector SDK, you can create environment-agnostic and reusable Connector runtime behavior.
+This section outlines how to set up a Connector project, test it, and run it locally.
+
+### Setup
+
+When developing a new Connector, we recommend using our
+[custom Connector template repository on GitHub](https://github.com/camunda/connector-template).
+This template is a [Maven](https://maven.apache.org/)-based Java project, and can be used in various
+ways such as:
+
+- _Create your own GitHub repository_: Click **Use this template** and follow the prompted steps.
+  You can manage code changes in your new repository afterward.
+- _Experiment locally_: Check out the source code to your local machine using [Git](https://git-scm.com/).
+  You won't be able to check in code changes to the repository due to restricted write access.
+- _Fetch the source_: Download the source code as a ZIP archive using **Code** > **Download ZIP**.
+  You can adjust and manage the code the way you like afterward, using your chosen source code
+  management tools.
+
+To manually set up your Connector project, include the following dependency to use the SDK.
+Ensure you adhere to the project outline detailed in the next section.
+
+<Tabs groupId="dependency" defaultValue="maven" values={
+[
+{label: 'Maven dependency', value: 'maven' },
+{label: 'Gradle dependency', value: 'gradle' }
+]
+}>
+
+<TabItem value='maven'>
+
+```xml
+<dependency>
+  <groupId>io.camunda.connector</groupId>
+  <artifactId>connector-core</artifactId>
+  <version>0.1.0</version>
+</dependency>
+```
+
+</TabItem>
+
+<TabItem value='gradle'>
+
+```yml
+implementation 'io.camunda.connector:connector-core:0.1.0'
+```
+
+</TabItem>
+</Tabs>
+
+### Project outline
+
+There are multiple parts of a Connector that enables it for reuse, as a
```

Comment:
Do "There are multiple parts of a Connector that enables it for reuse, as a reusable building block" mean the same thing, or could we just say either "There are multiple parts of a Connector that enables it for reuse" or "There are multiple parts of a Connector that enables it as a reusable building block"?

Reply:
Lets make it simple, I like the proposal `There are multiple parts of a Connector that enable it for reuse`